docker-compose.dev.yml

```diff
@@ -1,75 +1,57 @@
 services:
-    core:
-        build:
-            context: .
-            args:
-                - PACKAGE=core
-        container_name: core
-        env_file: ./.env.docker-compose
-        environment:
-            #s non secret
-            - ASSETS_BUCKET_NAME=assets.byron.pubpub.org
-            - MAILGUN_SMTP_HOST=smtp.mailgun.org
-            - MAILGUN_SMTP_PORT=465
-            - MAILGUN_SMTP_USERNAME=omitted
-            - OTEL_SERVICE_NAME=core.core
-            - PGDATABASE=postgres
-            - PGHOST=host.docker.internal
-            - PGPORT=54322
-            - PGUSER=postgres
-            - PGPASSWORD=postgres
-            - PUBPUB_URL=http://localhost:8080
-        networks:
-            - app-network
-        ports:
-            - "30000:3000"
-
-    core-nginx:
-        build: ./infrastructure/nginx
-        container_name: core-nginx
-        environment:
-            - NGINX_LISTEN_PORT=8080
-            - NGINX_PREFIX=/
-            - NGINX_UPSTREAM_HOST=core
-            - NGINX_UPSTREAM_PORT=3000
-            - OTEL_SERVICE_NAME=core.nginx
-        depends_on:
-            - core
```

Comment:
i removed the `core` and `jobs` build services for `.dev`. I never use them, I don't think any of you use them